src/lib/enum.js

```diff
@@ -194,6 +194,13 @@ const TransferState = {
   RESERVED_TIMEOUT: 'RESERVED_TIMEOUT'
 }
 
+const TransferStateEnum = {
```

Comment:
whats the difference between this and the TransferState constant above this? Should these no be added to that enum?

Reply:
They are not the same.

`TransferState` is the internal state representation where `TransferStateEnum` is the "Mojaloop" specification enumerated states.